Submitted By: Zack Winkles (winkie@linuxfromscratch.org)
Date: 2004-04-12
Initial Package Version: 1.1.0
Upstream Status: Not Submitted
Origin: Self-created
Description: Fixes compilation problems with GCC 3.4

diff -Naur imlib2-1.1.0.orig/src/rgbadraw.c imlib2-1.1.0/src/rgbadraw.c
--- imlib2-1.1.0.orig/src/rgbadraw.c	2003-07-12 21:21:23.000000000 -0400
+++ imlib2-1.1.0/src/rgbadraw.c	2004-04-12 18:59:23.883739546 -0400
@@ -2358,6 +2358,7 @@
                        if (ps)
                           ps->next = s;
                      nospans:
+			((void)0);
                     }
                }
              if (i == y2)
@@ -2365,6 +2366,7 @@
              i += step;
           }
       nolines:
+	  ((void)0);
      }
    for (i = 0; i < h; i++)
      {
